src/Azure/Shared/Storage/AzureTableDataManager.cs

```diff
@@ -708,7 +719,7 @@ private CloudTableClient GetCloudTableOperationsClient()
                 CloudStorageAccount storageAccount = AzureTableUtils.GetCloudStorageAccount(ConnectionString);
                 CloudTableClient operationsClient = storageAccount.CreateCloudTableClient();
                 operationsClient.DefaultRequestOptions.RetryPolicy = AzureTableDefaultPolicies.TableOperationRetryPolicy;
-                operationsClient.DefaultRequestOptions.ServerTimeout = AzureTableDefaultPolicies.TableOperationTimeout;
+                operationsClient.DefaultRequestOptions.ServerTimeout = this.tableOperationTimeout;
```

Comment:
It would also be useful to control the MaximumExecutionTime property here (the client-side timeout).

Reply:
Good point. Based on the retry guidance: https://docs.microsoft.com/en-us/azure/architecture/best-practices/retry-service-specific, it's better to configure the MaxExecutionTime instead.